cfgov/v1/tests/models/test_handlers.py

```diff
@@ -0,0 +1,72 @@
+import mock
+from collections import OrderedDict
+from unittest import TestCase
+from ...models.handlers import Handler, JSHandler
+
+
+class TestHandler(TestCase):
+    def setUp(self):
+        self.page = mock.Mock()
+        self.request = mock.Mock()
+        self.handler = Handler(self.page, self.request)
+
+    def test_process_raises_NotImplementedError(self):
+        with self.assertRaises(NotImplementedError) as nie:
+            self.handler.process({})
+
+    def test_get_streamfield_blocks_returns_list(self):
+        assert type(self.handler.get_streamfield_blocks()) is list
```

Comment:
I cannot find `assertIsInstance`. [Here's the mock class.](https://docs.python.org/3/library/unittest.mock.html#the-mock-class)